package/MDAnalysis/coordinates/GRO.py

```diff
@@ -357,31 +353,24 @@ def write(self, obj):
            *resName* and *atomName* are truncated to a maximum of 5 characters
         .. versionchanged:: 0.16.0
            `frame` kwarg has been removed
-        .. deprecated:: 1.0.0
```

Comment:
I think we’re meant to keep the old tag and just keep appending new stuff

Reply:
This came up before in: https://github.com/MDAnalysis/mdanalysis/pull/2494#discussion_r374805918 and I think the idea at the time was to remove deprecation notices since they get replaced by the versionchanged.

Reply:
I am glad that you have a good memory @IAlibay ! 

I added a section under [Writing Documentation: Documenting changes](https://github.com/MDAnalysis/mdanalysis/wiki/WritingDocumentation#documenting-changes) to capture what I believe is our current consensus. It's up for discussion, of course.

@lilyminium I know that the User Guide copied most of the wiki style guide so I created PR https://github.com/MDAnalysis/UserGuide/pull/76 with these changes.